frontend/src/metabase/query_builder/components/DataSelector/DataSelectorSchemaPicker/DataSelectorSchemaPicker.tsx

```diff
@@ -36,7 +34,7 @@ const DataSelectorSchemaPicker = ({
   ];
 
   return (
-    <Container>
+    <Box w={300}>
```

Comment:
This `300` comes up a lot in this PR. Maybe we should have a constant for it, WDYT?

Reply:
added a constant and reused it in DataSelector folder